src/coffee-script.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
```

Comment:
Yeah, would be nice though to somewhere test that cache files are written and files with the same content are only compiled once.

Basically, what coffeestack was testing now that that logic is being done here.